README.md

```diff
@@ -13,46 +13,48 @@ access Microsoft Cloud service resources. This version of the connector only sup
 information of a specific account (currently logged-in user).
  
 For more information, go to the module(s).
-- [ballerinax/microsoft.teams](https://docs.central.ballerina.io/ballerinax/microsoft.teams/0.1.0)
+- [microsoft.teams](teams/Module.md)
```

Comment:
Remove unnecessary spaces in the above paragraph

Reply:
Fixed